api_docs/videos.md

```diff
@@ -179,6 +179,7 @@ video[images_attributes][][caption] | Caption of the image | String
 video[images_attributes][][attachment] | Image as a file attachment
 video[images_attributes][][layout] | Must be one of 'poster' or 'square'. Default is 'landscape' | String
 video[images_attributes][][_destroy] | Pass in 'true' if you wish to remove an image | String
+video[custom_attributes][][attribute_name] | List of custom attributes with values that you wish to update on a given video | Array
```

Comment:
@harold29 Just a minor thing, `custom_attributes` is actually a Hash, it's a key-value pair of metadata.